components/drop-zone/provider.js

```diff
@@ -76,10 +71,6 @@ class DropZoneProvider extends Component {
 		} );
 	}
 
-	onDragOver( event ) {
-		event.preventDefault();
-	}
-
 	toggleDraggingOverDocument( event ) {
```

Comment:
This function is called so many times when dragging right now, do you think we should throttle it?

Reply:
Indeed. Great idea Riad. I am looking into this, currently trying something out with lodash. Sorry for the delayed response, I was just away since yesterday.

Reply:
Hey Riad, I've finally added throttling at 300ms using lodash. It feels ok to me at this value, but please have a go and let me know what you think. Also, this is the best I could come up with code-wise to get the `event.preventDefault` call working. Tested it in Chrome/Firefox/Safari/IE11. :-)